fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -2550,4 +2550,8 @@ public class Config extends ConfigBase {
 
     @ConfField(mutable = true)
     public static boolean replan_on_insert = false;
+
+    @ConfField(mutable = true, comment = "Whether or not resolve the fqdn of compute node/backend" +
```

Comment:
may not worth to add configuration. make the default behavior to always generating the ip address from fqdn?